test/models/stacks_test.rb

```diff
@@ -262,6 +271,17 @@ def setup
       assert @stack.active_task?
     end
 
+    test ".run_deploy_in_foreground triggers a deploy" do
+      stack = Stack.create!(repo_owner: 'foo', repo_name: 'bar', environment: 'production')
+      commit = shipit_commits(:first)
+      stack.commits << commit
+
+      Stack.any_instance.stubs(:trigger_deploy).with(anything, is_a(AnonymousUser), env: [], force: true, skip_queue: true)
+
+      Stack.run_deploy_in_foreground(stack: "#{stack.repo_owner}/#{stack.repo_name}/#{stack.environment}",
+                                     revision: commit.sha)
+    end
```

Comment:
It seems like this test is missing an assertion?

Reply:
see the other comment